import-service/src/importFileParser.js

```diff
@@ -0,0 +1,57 @@
+const AWS = require("aws-sdk");
+const csv = require("csv-parser");
+
+const BUCKET_NAME = "import-product-service";
+
+const importFileParser = async (event) => {
+  const { key } = event.Records[0]["s3"]["object"];
```

Comment:
it could be multiple files i suppose, you could use `forEach` for records to move all files